policies/templates/gcp_iam_restrict_service_account_creation_v1.yaml

```diff
@@ -0,0 +1,69 @@
+# Copyright 2019 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
```

Comment:
can we add a top level summary comment to explain what this template is trying to constrain?

Reply:
I checked other templates, there is no such comment. The naming of the constraint is self explanatory.